src/components/ChatList/GroupChatChannelItem.tsx

```diff
@@ -59,7 +60,7 @@ const GroupChatChannelItem = (props: ChannelItemProps) => {
 
   return (
     <CustomPressable onPress={onChannelPressed}>
-      <View style={styles.channelContainer}>
+      <View style={[styles.channelContainer, {backgroundColor: COLORS.signed_secondary}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color `COLORS.signed_secondary` is used for the background color of all channel types. This contradicts the PR description which states that different colors should be used for different channel types. Consider using a function or mapping to determine the appropriate color based on the `channelType`.

```diff
- <View style={[styles.channelContainer, {backgroundColor: COLORS.signed_secondary}]}>
+ <View style={[styles.channelContainer, {backgroundColor: getBackgroundColor(channelType)}]}>
```

In the above code, `getBackgroundColor` is a hypothetical function that returns the appropriate color based on the `channelType`. You would need to implement this function according to your specific requirements and color mappings.


<!-- This is an auto-generated comment by OSS CodeRabbit -->